src/shrub/v3/shrub_service.py

```diff
@@ -1,8 +1,132 @@
 """Service for working with shrub."""
+
 import yaml
 from pydantic import BaseModel
 
 
+class ConfigDumper(yaml.SafeDumper):
+    # Represent multiline strings in the form:
+    #     key: |
+    #       multiline string
+    #       multiline string
+    #       multiline string
+    def represent_scalar(self, tag, value, style=None):
+        if isinstance(value, str) and "\n" in value:
+            style = "|"
+        return super().represent_scalar(tag, value, style)
+
+    # Prefer using double quotes when able.
+    def analyze_scalar(self, scalar):
+        res = super().analyze_scalar(scalar)
+        if res.allow_single_quoted and res.allow_double_quoted:
+            res.allow_single_quoted = False
+        return res
```

Comment:
Note: this is a style preference. Single quotes (or both) may be preferred instead.